transformStream.js

```diff
@@ -3,9 +3,25 @@ const {Transform}=require("stream")
 
 const stringTr = new Transform({
     transform(chunk, encoding, callback) {

```

Comment:
Let's try to use `objectMode:true`config in order to work with chunks as JS object and get rid of JSON.parse and stringify.